Source/WebKit/WebProcess/WebProcess.cpp

```diff
@@ -687,6 +687,7 @@ void WebProcess::setIsInProcessCache(bool isInProcessCache)
 
     updateProcessName(IsInProcessInitialization::No);
     accessibilityRelayProcessSuspended(isInProcessCache);
+    completionHandler();
```

Comment:
You're failing to call the completion handler if the `#else` case. We should probably just move this out of the `#if / #else` scope.